db/blob/blob_file_builder_test.cc

```diff
@@ -108,7 +109,7 @@ class BlobFileBuilderTest : public testing::Test {
     ASSERT_EQ(footer.expiration_range, ExpirationRange());
   }
 
-  MockEnv mock_env_;
```

Comment:
Any essential need to change to std::unique_ptr<Env>?

Reply:
It allows me to make the MockEnv constructor private.  The MockFileSystem requires a Clock at the time it is being created, which requires two steps.